content/reference/cli/supported-commands.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cli Commands"
+---
+
+# Cli Commands
+
+## Setup and Config
+
+| Component | Supported  | Notes and limitations                   |
+|:----------|:-----------|:----------------------------------------|
+| `config`  | ✅          | planned support for `dolt config alias` |
+
+## Getting and Creating Projects
+
+| Component | Supported  | Notes and limitations |
+|:----------|:-----------|:----------------------|
+| `init`    | ✅          |                       |
+| `clone`   | ✅          |                       |
+
+## Basic Snapshotting
+
+| Component | Supported | Notes and limitations |
+|:----------|:----------|:----------------------|
+| `add`     | ✅         |                       |
+| `status`  | ✅         |                       |
+| `commit`  | ✅         |                       |
+| `notes`   | ❌         |                       |
+| `restore` | ❌         |                       |
+| `reset`   | ✅         |                       |
+| `rm`      | ✅         |                       |
+
+## Branching and Merging
+
+| Component  | Supported | Notes and limitations                                       |
+|:-----------|:----------|:------------------------------------------------------------|
+| `branch`   | ✅         |                                                             |
+| `checkout` | ✅         |                                                             |
+| `merge`    | ✅         |                                                             |
+| `log`      | 🟠        | planned support for `dolt log --all` and `dolt log --graph` |
```

Comment:
I think it's fine to call this one supported. The flags you mention in the nots and limitations is sufficient to say it's not 100%.